apps/meteor/server/services/translation/service.ts

```diff
@@ -28,9 +28,22 @@ export class TranslationService extends ServiceClassInternal implements ITransla
 		return this.translateText(text, language);
 	}
 
-	async translateToServerLanguage(text: string): Promise<string> {
+	async translateToServerLanguage(text: string, args?: Record<string, string>): Promise<string> {
 		const language = await this.getServerLanguageCached();
 
-		return this.translateText(text, language);
+		return this.translateText(text, language, args);
+	}
+
+	async translateMultipleToServerLanguage(keys: string[]): Promise<Array<{ key: string; value: string }>> {
+		const language = await this.getServerLanguageCached();
+
+		return Promise.all(
+			keys.map(async (key) => {
+				return {
+					key,
+					value: await this.translateText(key, language),
+				};
+			}),
+		);
```

Comment:
Instead of this being a promise, this can call i18n directly and avoid the promises. wdyt?